CETEN-OpenBar/bar
# Feature/PizzaPriceRestock

- add a boolean special price for categories
- if this special price is true then during the restock the all price will be Ceten price except for externe which will be externe

how to test this pr:

- go to admin/categories
- check a special price checkbox for a category
- go to panel/products/reappro
- add a product which is in the category you check (has special price) and one not in a category with special price
- go to admin/produits
- check that the price of the item with special price are correct (all price will be Ceten price except for externe which will be externe) and the other 

Special case handle:
- if a item don't have a category it give a one by default with "basic" price
